action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
   melos-version:
     description: 'The Melos version to make available on the path'
     required: false
+  dart-version:
+    description: 'The dart version to pass to dart-lang/setup-dart action used for publishing, this helps to avoid race condition when having flutter SDK setup with the version different from the latest dart'
```

Comment:
If this is longer than 80 you can break it up.
(On my phone, so I can't check easily)